src/runtime-rs/crates/runtimes/src/shim_mgmt/handlers.rs

```diff
@@ -50,3 +55,17 @@ async fn agent_url_handler(
         .unwrap_or_else(|_| String::from(""));
     Ok(Response::new(Body::from(agent_sock)))
 }
+
+async fn direct_volume_stats_handler(
+    _sandbox: Arc<dyn Sandbox>,
+    _req: Request<Body>,
+) -> Result<Response<Body>> {
+    Ok(Response::new(Body::from("is sandbox fully implemented?")))
```

Comment:
`Direct volume` depend on the functionality of block device hotplug. This pr is #5441 is doing this.